src/components/Error/index.js

```diff
@@ -0,0 +1,3 @@
+import { Error } from './Error';
+
+export { Error };
```

Comment:
You should be able to do `export { Error } from './Error';` if you want

Reply:
Neat! I got this trick from a post somewhere and didn't think much about improving it. I'll change the index files soon.